controllers/controllers_test.go

```diff
@@ -258,9 +259,16 @@ func (r *fakeSeedsResolver) ResolveSeedEndpoints(ctx context.Context, dc *cassdc
 	return r.callback(dc)
 }
 
+type fakeManagementApiFactory struct {
+}
+
+func (f fakeManagementApiFactory) NewManagementApiFacade(ctx context.Context, dc *cassdcapi.CassandraDatacenter, k8sClient client.Client, logger logr.Logger) (cassandra.ManagementApiFacade, error) {
```

Comment:
minor nit - I think `ctx` should remain an argument in each of the methods in the interface. I am fine addresses separately if we decide to change it.